plugins/confluence-plugin/src/components/PageContent.tsx

```diff
@@ -0,0 +1,71 @@
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
+import React, { useState, useEffect } from "react";
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
+// import React from "react";
+import { isEmpty, isNil } from "lodash";
+import "../baseStyles.css";
+import {
+  Box,
+  Text,
+  usePluginContext,
+} from "@cortexapps/plugin-core/components";
+import { getEntityYaml } from "../api/Cortex";
+import { getConfluenceDetailsFromEntity } from "../lib/parseEntity";
+
+const baseJIRAUrl = "https://cortex-se-test.atlassian.net";
+
+const PageContent: React.FC = () => {
+  const [pageContent, setPageContent] = useState<string | undefined>();
+  const [pageTitle, setPageTitle] = useState<string | undefined>();
+  const [entityPage, setEntityPage] = useState<any | string>();
+
+  const context = usePluginContext();
+
+  useEffect(() => {
+    const fetchEntityYaml = async (): Promise<void> => {
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      const entityTag = context.entity?.tag;
+      if (!isNil(entityTag)) {
+        try {
+          const yaml = await getEntityYaml(context.apiBaseUrl, entityTag);
+          const pageID = isEmpty(yaml)
+            ? undefined
+            : getConfluenceDetailsFromEntity(yaml);
+          setEntityPage(pageID?.pageID);
+          const jiraURL =
+            baseJIRAUrl +
+            "/wiki/rest/api/content/" +
+            // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
+            pageID?.pageID +
```

Comment:
eslint is making a valid complaint here saying that `pageID?.pageID` can be `undefined`, in which case this will evaluate to the string `"undefined"`. I think it would be cleanest to handle this scenario explicitly
```suggestion
          if (!pageID?.pageID) {
            throw new Error('No Confluence details for entity')
          }
          
          setEntityPage(pageID.pageID);
          
          const jiraURL = `${baseJIRAUrl}/wiki/rest/api/content/${pageID.pageID}?expand=body.view`;
```
Also, we can use string interpolation to make the URL building a bit cleaner